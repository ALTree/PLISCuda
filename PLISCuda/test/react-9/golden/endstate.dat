subvolumes=9
species=3
time=8.387568

0 0 100 
0 0 98 
0 0 105 
0 0 92 
0 0 101 
0 0 99 
0 0 106 
0 0 98 
0 0 101 
