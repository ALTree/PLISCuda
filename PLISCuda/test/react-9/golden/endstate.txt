subvolumes=9
species=3
time=8.135010

0 0 98 
0 0 101 
0 0 98 
0 0 99 
0 0 105 
0 0 97 
0 0 100 
0 0 98 
0 0 104 
